qa/suites/rbd/migration-external/1-base/install.yaml

```diff
@@ -0,0 +1,8 @@
+meta:
+- desc: run two ceph clusters
+tasks:
+- install:
+- ceph:
+- ceph:
+    cluster: cluster2
```

Comment:
Make the other cluster `cluster1` for symmetry and so that it's clear what is run where.

Reply:
I tried the proposed simple change to `run_qemu`.
It didn't work as the qemu config did not have this property set.
The qemu config actually expects sub-configs `{client_name: client_config}`.
We can define a new `cluster` property to the `client_config`.
However, this will require widespread changes for consumers of this dictionary to support it.
For example, it will require changing the functions: `generate_iso`, `create_images`, `create_clones`, `create_encrypted_devices`, as well as functions in the `rbd` task: `create_image`, `clone_image`, `dev_create`, and perhaps also `xfstests` (for config consistency).

Would you still like to make this change is part of this PR?

Reply:
> The qemu config actually expects sub-configs `{client_name: client_config}`.
> We can define a new `cluster` property to the `client_config`.

It would be much better to follow what is done in the `workunit` task and make use `teuthology.split_role()`:
```
        - workunit:
            clients:
              backup.client.0: [foo] # cluster is 'backup'
              client.1: [bar] # cluster is implicitly 'ceph'
```
> However, this will require widespread changes for consumers of this dictionary to support it.
> For example, it will require changing the functions: `generate_iso`, `create_images`, `create_clones`, `create_encrypted_devices`, as well as functions in the `rbd` task

Yes, it's a number of places to change but the changes aren't widespread at all.  For the `rbd` task, it should mostly be a matter of adding `--cluster <cluster_name>` to `rbd create`, `rbd rm`, etc commands.  Some of the other functions that you have listed, like `generate_iso()`, shouldn't need changes at all.  You have done pretty much the same thing in https://github.com/ceph/ceph/pull/40363 when propagating `encryption_format` to `create_image()`, `clone_image()`, etc ;)

> Would you still like to make this change is part of this PR?

I would like to see them as part this feature -- otherwise it's yet another item on the ever growing backlog.  Whether in this PR or split into a prerequisite PR to reduce the number of moving pieces here, either is fine with me.